package day30;

public class TryCatc01 {

	public static void main(String[] args) {
		//try'dan sonraistedigimiz kadar catch kulalanabiliriz
		//sartimiz yukardaki catch herzaman alttakinin child'i olmali
		
	int num1=12;
	int num2=0;
	try {
		System.out.println(num1/num2);
	} catch (ArithmeticException e) {
		System.out.println("sifir ile b√∂lme yapilmaz");
		
	}finally {
		System.out.println("Aferin sana");
	}
		
		
		
		
		
		
	}

}
